database/init.sql

```diff
@@ -78,7 +80,7 @@ CREATE TABLE appointment (
 CREATE TABLE unavailable (
 	id SMALLINT UNSIGNED NOT NULL AUTO_INCREMENT,
 	date_id SMALLINT UNSIGNED NOT NULL,
-	time_id SMALLINT UNSIGNED NOT NULL,
+	time TIME NOT NULL,
 	branch_id SMALLINT UNSIGNED NOT NULL,
 	manager_id SMALLINT UNSIGNED NOT NULL,
 	service_id SMALLINT UNSIGNED NOT NULL,
```

Comment:
So if branch_id, manager_id or service_id are 0 (Since they can't be null), then that means the unavailable time will apply to all branches, all managers, or all services? And if the date_id (calendar_id) is 0, then it will apply to all days? I forget how we were going to make something unavailable for every week. We would need a day_of_week column for that to work.

Reply:
Maybe we could change manager, branch and service so they can be null in this table. The date id should never be 0, it will start at 1. Once I add the calendar table I will add a day of week column into this table as well.

Reply:
Okay that sounds good. So a null for any of those id's will mean it applies to all.